js/parallax.js

```diff
@@ -39,7 +39,7 @@
         $this.children("img").one("load", function() {
           updateParallax(true);
         }).each(function() {
-          if(this.complete) $(this).load();
+          if(this.complete) updateParallax(true);
```

Comment:
what if you change 
```
trigger('load');
```
to
```
trigger('load')
```
?
That's what the jquery 3 migration guide recommends

https://jquery.com/upgrade-guide/3.0/#breaking-change-load-unload-and-error-removed